### PR TITLE
finished challenge

### PR DESCRIPTION
**Pull request by slakeyj**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code-challenges/fizzBuzzTree/fizz-buzz-tree.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    if (root.value % 3 === 0 && root.value % 5 === 0) {
+      root.value = 'FizzBuzz';
+    } else if (root.value % 3 === 0) {
+      root.value = 'Fizz';
+    } else if (root.value % 5 === 0) {
+      root.value = 'Buzz';
+    } else {
+      root.value = root.value.toString();
+    }
```
Comment (XiaojuanCai) [inline code-challenges/fizzBuzzTree/fizz-buzz-tree.js]: You changed the original tree. Can you modify it to generate a new fizzbuzztree without changing the original tree?

File: code-challenges/fizzBuzzTree/fizz-buzz-tree.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    collection.push(root.value);
+
+    if (root.right) {
+      inOrder(root.right);
+    }
+    return collection;
+
+  }
+
+  const fizzBuzzed = inOrder(tree.root)
```
Comment (XiaojuanCai) [inline code-challenges/fizzBuzzTree/fizz-buzz-tree.js]: What's the type of fizzBuzzed here? A tree or an array?

File: code-challenges/fizzBuzzTree/fizz-buzz-tree.test.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  it('turns the number into a string if the number is neither divisible by 3 or 5', () => {
+    binaryTree.insert(2);
+    binaryTree.insert(5);
+    binaryTree.insert(7);
+    binaryTree.insert(11);
+    binaryTree.insert(30);
+
+    const fizzBuzzIt = fizzBuzzTree(binaryTree);
+    expect(fizzBuzzIt).toEqual(['2', 'Buzz', '7', '11', 'FizzBuzz']);
+  })
```
Comment (XiaojuanCai) [inline code-challenges/fizzBuzzTree/fizz-buzz-tree.test.js]: Your unit tests are fantastic!
